src/main/java/techreborn/tiles/generator/TileSolarPanel.java

```diff
@@ -113,7 +117,7 @@ public boolean canProvideEnergy(final EnumFacing direction) {
 
 	@Override
 	public double getBaseMaxOutput() {
-		return maxOutput;
+		return powerToAdd;
```

Comment:
It should return output per tier, not actual values it is generating. We have internal buffer to supply diff between gen and output.

Reply:
I need more clarification, I don't quite understand what you are implying.

Reply:
getBaseMaxOutput should return theoretical maximum output of machine before any upgrades applied. So it should always be, for example 1024 FE\t for ultimate solar panel both day and night.

Reply:
I had it set as the current power generation because it showed on the tooltip, I guess I could change it back but then the players would get no indication of the current state until someone works out a alternative tooltip.

Reply:
Yes, none of machines are showing current generation yet. I hope one day this will be worked out )